composer.json

```diff
@@ -38,7 +38,8 @@
         "predis/predis": "^1.0",
         "snc/redis-bundle": "~1.1",
         "guzzle/plugin-cache": "3.9.2",
-        "badges/poser": "v1.2.2"
+        "badges/poser": "v1.2.2",
```

Comment:
This is the old composer.json file... 😔 maybe it's better if you close this PR and start over brannching off from `release/v3.0.0`. If so, please avoid including trivial changes (like removal of spaces in documentation)